src/modules/todolist.js

```diff
@@ -1,9 +1,11 @@
 /* eslint-disable import/no-cycle */
```

Comment:
You have done very well on this project 👍 However, the linter checks are in place to ensure coding best practices and also ensure that you don't encounter bugs or security issues when you scale up your project. I suggest you do not disable them but fix the issues that are pointed out. I can assure you that this is a fix that is worth it and very important. 💯